torch/distributed/rpc/__init__.py

```diff
@@ -104,18 +121,18 @@ def init_rpc(
                 raise TypeError(
                     f"Could not infer backend for options {rpc_backend_options}"
                 )
-            if backend != BackendType.TENSORPIPE:
+            if backend != BackendType.TENSORPIPE:  # type: ignore[attr-defined]
```

Comment:
Can you please elaborate, why `BackendType.TNEOSRPIPE` needs to be annotate with `ignore[attr-defined]`?

Reply:
It is because BackendType.TENSORPIPE is a dynamically constructed type, which mypy can't handle at this moment: https://github.com/python/mypy/issues/4865.